tests/core/test_external_search.py

```diff
@@ -4484,3 +4578,38 @@ def bulk(self, docs, **kwargs):
         assert work == record.obj
         assert True == record.transient
         assert "There was an error!" == record.exception
+
+
+class DBStatementCounter(object):
```

Comment:
Apparently we don't need to subclass `object` anymore in Python 3.

```
class DBStatementCounter:
```

Reply:
This is true, this is stackoverflow code for counting statements, let me rectify
https://stackoverflow.com/a/33691585